lib/SIL/Verifier/MemoryLifetimeVerifier.cpp

```diff
@@ -274,6 +282,30 @@ void MemoryLifetimeVerifier::requireBitsSet(const Bits &bits, SILValue addr,
   }
 }
 
+void MemoryLifetimeVerifier::requireBitsSetForArgument(const Bits &bits, SILValue addr,
+                                           SILInstruction *applyInst) {
+  // Optimizations can rely on alias analysis to know that an in-argument (or
+  // parts of it) is not actually read.
+  // We have to do the same in the verifier: if alias analysis says that an in-
+  // argument is not read, there is no need that the memory location is initialized.
+
+  // Not all calls to the verifier provide the alias analysis.
+  if (!aliasAnalysis)
```

Comment:
Out of curiosity, when don't we have the PassManager?  I found `verifyIncompleteOSSA`, `verifyCriticalEdges`, `SILVTable::verify`.  (If that's it, it might make sense to remove the default `= nullptr` from `SILPassManager *passManager = nullptr`.)

Reply:
There are several places in the optimizer where we call `function->verify()`.

Reply:
Makes sense.  The reason I was asking is that in all those places, the verifier now will skip verification of applies in all those places.